.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG]"
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+<!--
+A high-level description of what the bug is. Make sure to clearly and concisely describe where the bug was encountered, what happened and what you expected to happen. Provide more details on each of these in the sections below.
+-->
+
+**To Reproduce**
+<!--
+Provide the information necessary to reproduce the bug. If the bug occurred when executing ``skbase`` related code, provide a reproducible code example. If the bug arose in another context, provide instructions for recreating the bug.
+-->
+
+
+**Expected behavior**
+<!--
+Provide more detail on what you expected to happen.
+-->
+
+
+**Environment**
+<!--
+Please provide information about the environment you encountered the bug in. If your bug was related to running ``skbase`` Python code, provide details about your Python environment (e.g., operating system and version, version of Python, version of ``skbase``, version of other related packages, etc).
```

Comment:
is there an easier way, i.e., more instruction based, less language based, to ensure people provide the right information here? E.g., sth like please run `print_versions` or similar

Reply:
I'm going to leave this for later.